### PR TITLE
store/tikv: export LockResolver and TxnStatus.

### PR DESCRIPTION
**Pull request by disksing**
@iamxy These 2 functions are exported for binlog. PTAL @zimulala @coocood 

```
func NewLockResolver(etcdAddrs []string, clusterID uint64) (*LockResolver, error)

(lr *LockResolver) GetTxnStatus(txnID uint64, primary []byte) (TxnStatus, error)
```

### CODE REVIEW

File: store/tikv/lock_resolver.go [hunk 1]
```diff
@@ -29,25 +31,42 @@ type LockResolver struct {
...
 
-func (s txnStatus) isCommitted() bool { return s > 0 }
-func (s txnStatus) commitTS() uint64  { return uint64(s) }
+// TxnStatus represents a txn's final status. It should be Commit or Rollback.
+type TxnStatus uint64
+
+// IsCommitted returns true if the txn's final status is Commit.
+func (s TxnStatus) IsCommitted() bool { return s > 0 }
+
+// CommitTS returns the txn's commitTS. It is valid iff `IsCommitted` is true.
```
Comment (zimulala) [inline store/tikv/lock_resolver.go]: iff -> if ?

**Comment by shenli** [comment]
LGTM
PTAL @coocood

File: store/tikv/lock_resolver.go (hunk 1 above)
Comment (disksing) [inline store/tikv/lock_resolver.go]: I use it intentionally, which means if and only if :)

**Comment by zimulala** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Pull request merged**